src/components/frame/Extensions/layouts/ColonyLayout.tsx

```diff
@@ -91,17 +92,22 @@ const ColonyLayout: FC<PropsWithChildren> = ({ children }) => {
   const userHub = useMemo(() => <UserHubButton />, []);
 
   const getUserNavigation = useCallback(
-    (isHidden?: boolean) =>
-      !isTokensModalOpen ? (
-        <UserNavigationWrapper
-          txButtons={txButtons}
-          userHub={userHub}
-          isHidden={isHidden}
-          extra={
-            <>
-              <JoinButton />
-              {/* Hide Initially */}
-              {/* {!isActionSidebarOpen ? (
+    (isHidden?: boolean) => (
+      <UserNavigationWrapper
+        txButtons={txButtons}
+        userHub={userHub}
+        className={clsx(
+          'modal-blur-navigation [.show-header-in-modal_&]:z-[1001]',
+          {
+            'relative z-[11]': !isTablet,
```

Comment:
Is anything stopping us from using `z-20`?

Reply:
I just think that 11 is enough here. I haven't tested it with 20

Reply:
In this case, we could add these values (11 and 1001) in `tailwind.config.js` if the already defined values are not usable in this scenario